docs/schema-loader.md

```diff
@@ -92,6 +92,12 @@ Create/Delete schemas in the storage defined in the config file
                       The replication strategy, must be SimpleStrategy or
                         NetworkTopologyStrategy (supported in Cassandra)
       --ru=<ru>       Base resource unit (supported in DynamoDB, Cosmos DB)
+      --upgrade       Upgrades the ScalarDB environment to support the latest
+                        version of the ScalarDB API. Typically, you will be
+                        requested, as indicated on the release notes, to run
+                        this command after updating the ScalarDB version of
+                        your application environment.
```

Comment:
```suggestion
      --upgrade       Upgrades the ScalarDB environment to support the latest
                        version of the ScalarDB API. Typically, you will
                        need, as indicated on the release notes, to run
                        this command after updating the ScalarDB version of
                        your application environment.
```
It seems that we should update this description same as the following comment.
https://github.com/scalar-labs/scalardb/pull/1311/files#r1408451206

Reply:
I forgot to fix this one, thank you.